templates/pretalx.cfg.j2

```diff
@@ -31,9 +31,11 @@ port = {{ pretalx_mail_port }}
 tls = {{ pretalx_mail_tls }}
 ssl = {{ pretalx_mail_ssl }}
 
-{% if pretalx_celery %}[celery]
+{% if pretalx_celery %}
```

Comment:
Is there a reason for this change?

If you insist on this for readability, please use the appropriate Jinja markers to make sure the `pretalx.cfg` output does not change.

Reply:
Done, please re-review.